src/affine.py

```diff
@@ -555,15 +555,15 @@ def __matmul__(self, other):
 
         Apply the transform using matrix multiplication, creating
         a resulting object of the same type.  A transform may be applied
-        to another transform or vector array.
+        to another transform, a vector, vector array, or shape.
```

Comment:
This language is also restored. I wasn't really sure what the last two items were until this adaptation.